src/Dapr.Client/DaprClientBuilder.cs

```diff
@@ -14,19 +14,22 @@ namespace Dapr.Client
     /// </summary>
     public sealed class DaprClientBuilder
     {
-        const string defaultDaprGrpcPort = "50001";
-        string daprEndpoint;
-        JsonSerializerOptions jsonSerializerOptions;
-        GrpcChannelOptions gRPCChannelOptions;
-
+        private const string defaultDaprGrpcPort = "50001";
+        internal string daprEndpoint;
```

Comment:
As I understand this is for validation in tests. expose it as internal get property.